src/nft/components/collection/CollectionPageSkeleton.tsx

```diff
@@ -3,27 +3,37 @@ import { Column, Row } from 'nft/components/Flex'
 import { useIsMobile } from 'nft/hooks'
 import { CollectionBannerLoading } from 'nft/pages/collection'
 import { COLLECTION_BANNER_HEIGHT } from 'nft/pages/collection'
+import * as styles from 'nft/pages/collection/index.css'
+import styled from 'styled-components/macro'
 
 import { ActivitySwitcherLoading } from './ActivitySwitcher'
 import { CollectionNftsAndMenuLoading } from './CollectionNfts'
 import { CollectionStatsLoading } from './CollectionStats'
 
+const CollectionDescriptionSection = styled(Column)`
```

Comment:
nit / idea: Maybe we could try using Column/Row from `src/components`? I just found them, looks like they're direct equivalents of the Column/Box from Sprinkles that we're eventually going to get rid of. Might be a good practice to start replacing them as we refactor things, just like we move to Styled Components.

Reply:
seems to work pretty well